src/pkg/packager/load/load.go

```diff
@@ -53,6 +58,24 @@ func PackageDefinition(ctx context.Context, packagePath string, opts DefinitionO
 	if err != nil {
 		return v1alpha1.ZarfPackage{}, err
 	}
+
+	if len(pkg.Values.Files) > 0 && !feature.IsEnabled(feature.Values) {
+		return v1alpha1.ZarfPackage{}, fmt.Errorf("creating package with Values files, but \"%s\" feature is not enabled."+
+			" Run again with --features=\"%s=true\"", feature.Values, feature.Values)
+	}
+
+	// Load top-level values files and ensure they're merged with user-provided values.
+	// Resolve values file paths relative to the package path
+	valueFilePaths := make([]string, len(pkg.Values.Files))
+	for i, vf := range pkg.Values.Files {
+		valueFilePaths[i] = filepath.Join(packagePath, vf)
+	}
+	values, err := value.ParseFiles(ctx, valueFilePaths, value.ParseFilesOptions{})
+	if err != nil {
+		return v1alpha1.ZarfPackage{}, fmt.Errorf("failed to parse values files: %w", err)
+	}
+	opts.Values.DeepMerge(values)
```

Comment:
This execution of `DeepMerge` merges the package manifests values files onto the (assuming) provided command line values. 

`deploy.go` does the opposite and is what I assumed the correct stance. 

Is this intentional?